cfme/fixtures/v2v_fixtures.py

```diff
@@ -212,9 +212,14 @@ def vddk_url():
     return url
 
 
-def get_conversion_data(target_provider):
+def get_conversion_data(appliance, target_provider):
     if target_provider.one_of(RHEVMProvider):
-        resource_type = "ManageIQ::Providers::Redhat::InfraManager::Host"
+        resource_type = (
+            "ManageIQ::Providers::Redhat::InfraManager::Host"
+            if appliance.version < '5.11'
+            else "ManageIQ::Providers::Openstack::CloudManager::Vm"
```

Comment:
This looks wrong @nachandr 
    
       "ManageIQ::Providers::Openstack::CloudManager::Vm" 
Shouldn't it be 
          
       "ManageIQ::Providers::Redhat::InfraManager::Vm"

Please test this with UCI and then update the proper command